.circleci/config.yml

```
@@ -0,0 +1,94 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+jobs:
+  build-setup:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run:
+                name: Install Yarn package manager
+                command: curl -o- -L https://yarnpkg.com/install.sh | bash
+            - run:
+                name: Install javascript dependencies (using Yarn)
+                command: yarn install --ignore-engines
+  build-web:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run:
+                name: Build design tokens for web
+                command: yarn web
+  build-docs:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run:
+                name: Build design tokens for web
+                command: yarn docs
+            - persist_to_workspace:
+                root: dist
+                paths:
+                  - docs
+  deploy-docs:
+    docker:
+      - image: node:8.10.0
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - attach_workspace:
+                at: dist
+            - run:
+                name: Install gh-pages dependencies
+                command: |
+                  yarn global add gh-pages@2.0.1 --silent
+                  git config user.email "ops@useblu.com.br"
+                  git config user.name "Blu"
+            - run:
+                name: Deploy docs to gh-pages branch
+                command: gh-pages --dist dist/docs
+
+workflows:
+  nightly:
+    triggers:
+      - schedule:
+          cron: '0 00,01,02,03,04,05,06,07,08,18,19,20,21,22,23 * * *'
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-setup
+      - build-web:
+          requires:
+            - build-setup
+      - build-docs:
+          requires:
+            - build-setup
+      - deploy-docs:
+          requires:
+            - build-setup
+            - build-docs
+  build-deploy:
+    jobs:
+      - build-setup
+      - build-web:
+          requires:
+            - build-setup
+      - build-docs:
+          requires:
+            - build-setup
+      - deploy-docs:
+          requires:
+            - build-setup
+            - build-docs
```

Change to trigger deploy-docs job only in master branch

```suggestion
  build:
    jobs:
      - build-setup
      - build-web:
          requires:
            - build-setup
      - build-docs:
          requires:
            - build-setup
      - deploy-docs:
          requires:
            - build-docs
          filters:
            branches:
              only: master
```